src/node/util.ts

```diff
@@ -58,10 +58,10 @@ export const paths = getEnvPaths()
  * On MacOS this function gets the standard XDG directories instead of using the native macOS
  * ones. Most CLIs do this as in practice only GUI apps use the standard macOS directories.
  */
-export function getEnvPaths(): Paths {
+export function getEnvPaths(platform = process.platform): Paths {
```

Comment:
✍🏼 This change was suggested by @code-asher 🔥

By passing in `process.platform`, we can override it in our tests without having to do hacky stuff like modify `process.platform` (which I was doing before).